corehq/couchapps/export_instances_by_is_daily_saved/views/view/map.js

```diff
@@ -1,5 +1,5 @@
 function(doc) {
     if (doc.doc_type === "FormExportInstance" || doc.doc_type === "CaseExportInstance") {
-        emit([doc.is_daily_saved_export, doc.domain, doc.doc_type], null);
+        emit([doc.is_daily_saved_export, doc.auto_rebuild_enabled, doc.last_accessed, doc.domain, doc.doc_type], null);
```

Comment:
`domain` and `doc_type` are pretty useless at the end of the key, might as well remove them